csireverseproxy/main.go

```diff
@@ -396,33 +402,14 @@ func (s *Server) EventHandler(k8sUtils k8sutils.UtilsInterface, secret *corev1.S
 	}
 	found = conf.IsSecretConfiguredForArrays(secret.Name)
 	if found {
-		if getEnv(common.EnvReverseProxyUseSecret, "false") == "true" {
```

Comment:
This code not needed as we never invoke the event handler for mounted secret. Mounted secrets are only monitored via viper and not directly as K8S entities.